### ISSUE TITLE
preprocessors

### ISSUE THREAD

**Issue by jots** [Webstrates/Codestrates#6]
Is there an ability to use preprocessors so that code can be written in coffeescript, css written in less/scss etc?

**Comment by raedle** [comment]
@jots Yes, we have successfully integrated the Typescript and SCSS transpilers into Codestrates. It's in our experimental version of Codestrates, though. We will release it in the near future. Contact me via email if you'd like pre-access
